odoo/addons/test_lint/tests/_odoo_checker_gettext.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+import astroid
+from pylint import checkers, interfaces
+
+
+class OdooBaseChecker(checkers.BaseChecker):
+    __implements__ = interfaces.IAstroidChecker
+    name = 'odoo'
+
+    msgs = {
+        'E8502': (
+            'Bad usage of _ method.',
+            'gettext-variable',
+            'See https://www.odoo.com/documentation/13.0/reference/translations.html#variables'
+        )
+    }
+
+    def _get_func_name(self, node):
+        func_name = (
+            isinstance(node.func, astroid.Name) and node.func.name or
+            isinstance(node.func, astroid.Attribute) and node.func.attrname)
```

Comment:
What use-case do you want to support?  Calling `self._(...)`?  Or maybe `odoo._(...)`?  Is that compliant with message extraction?  If you cannot extract the message, it should be flagged as an error.